src/connection.ts

```diff
@@ -191,6 +189,7 @@ export class Connection {
       this._activeHost = (this._activeHost + 1) % this._hosts.length;
     }
     this._activeTasks += 1;
+    // @ts-ignore
```

Comment:
why do we use ts-ignore here and also can you add the rule in the comment after ts-ignore

Reply:
Typescript changed a lot from 2.6.2 to 4.9.4 we can't use `task.options` in this way. Maybe we can refactor this part of the code, but it is perhaps not possible until we fix all integration tests.